cypress-tests/cypress/e2e/PaymentMethodListTest/00000-PaymentMethodListTests.cy.js

```diff
@@ -599,4 +600,85 @@ describe("Payment Method list using Constraint Graph flow tests", () => {
       });
     }
   );
+
+  context(
+    `
+    MCA1 -> Stripe configured with credit = { currency = "USD" }\n
+    MCA2 -> Novalnet configured with credit = { currency = "EUR" }\n
+    Payment is done with currency as as USD and no billing address\n
+    The resultant Payment Method list should only have credit with stripe\n
+    `,
+    () => {
+      before("seed global state", () => {
+        cy.task("getGlobalState").then((state) => {
+          globalState = new State(state);
+        });
+      });
+
+      it("merchant-create-call-test", () => {
+        cy.merchantCreateCallTest(fixtures.merchantCreateBody, globalState);
+      });
+
+      it("api-key-create-call-test", () => {
+        cy.apiKeyCreateTest(fixtures.apiKeyCreateBody, globalState);
+      });
+      it("customer-create-call-test", () => {
+        cy.createCustomerCallTest(fixtures.customerCreateBody, globalState);
+      });
+      // stripe connector create with card credit enabled in USD
+      it("connector-create-call-test", () => {
+        cy.createNamedConnectorCallTest(
+          "payment_processor",
+          fixtures.createConnectorBody,
+          cardCreditEnabledInUsd,
+          globalState,
+          "stripe",
+          "stripe_US_default"
+        );
+      });
+
+      // novalnet connector create with card credit enabled in EUR
+      it("connector-create-call-test", () => {
+        cy.createNamedConnectorCallTest(
+          "payment_processor",
+          fixtures.createConnectorBody,
+          cardCreditEnabledInEur,
+          globalState,
+          "novalnet",
```

Comment:
are the api keys updated in the creds file? if not, pml tests are bound to fail.

Reply:
yes!

Reply:
just checked, no it is not.

Reply:
Sure checking! I assumed you meant the local creds.json